app/main.py

```diff
@@ -16,4 +16,20 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [], "immutable": []}
+
+list_of_names = [
```

Comment:
In this task you just need to fill in the dictionary manually, in the next one you will use algorithms